docs/UserGuide/Operation Manual/DML Data Manipulation Language.md

```diff
@@ -414,10 +416,11 @@ Example 1: get the last point of root.ln.wf01.wt01.speed:
 |  5   | root.ln.wf01.wt01.speed | 100   |
 ```
 
-Example 2: get the last speed, status and temperature points of root.ln.wf01.wt01
+Example 2: get the last speed, status and temperature points of root.ln.wf01.wt01,
+whose timestamp in range [5, 10).
 
 ```
-> select last speed, status, temperature from root.ln.wf01.wt01
```

Comment:
Hm.. I think the changes here have no relation to this PR.

Reply:
Yes..that may caused by my misoperation on Git. I will roll back the version, submit again.